.CI/update_token.py

```diff
@@ -0,0 +1,198 @@
+from __future__ import print_function
```

Comment:
Would it be possible to use the same functionality from `conda-smithy` or are there some issues with that approach?

Reply:
I copied over @jjhelmus's code, but why not? I'll update to use conda-smithy

Reply:
Makes sense. Ok. If you do run into issues using `conda-smithy`, it would be good to know what they are so we can fix them.

Reply:
Done